test/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ celeritas_add_library(testcel_harness
 target_compile_features(testcel_harness PUBLIC cxx_std_14)
 celeritas_target_link_libraries(testcel_harness
   PUBLIC Celeritas::corecel GTest::GTest
+  PRIVATE ${nlohmann_json_LIBRARIES}
```

Comment:
Is it really okay to have `PRIVATE` followed by nothing?
What if later on somebody add something after:
```suggestion
  PRIVATE ${nlohmann_json_LIBRARIES}
  OTHERTAG ${othervalues}
```
?

Reply:
I tested before pushing and it does work as expected.

Reply:
Also besides public and private what other tags go there?

Reply:
I didn't flip the lines, I tested the code that's in there. With JSON unavailable, the current code works.

Reply:
Fair enough ...